zebrad/src/components/mempool/storage/verified_set.rs

```diff
@@ -24,12 +26,15 @@ use super::super::SameEffectsTipRejectionError;
 #[derive(Default)]
 pub struct VerifiedSet {
     /// The set of verified transactions in the mempool.
-    transactions: VecDeque<UnminedTx>,
+    transactions: VecDeque<VerifiedUnminedTx>,
 
     /// The total size of the transactions in the mempool if they were
     /// serialized.
     transactions_serialized_size: usize,
 
+    /// The total cost of the verified transactons in the set.
+    total_cost: u32,
```

Comment:
Security Risk: Let's match the type of `cost()` here.

Adding `u64`s to a `u32` total increases the risk of silent truncation in release code.
(That's unlikely due to our size limits and parsing algorithm, but let's just be safe!)

```suggestion
    total_cost: u64,
```

Reply:
I think addressed in 6fe52368, all `cost` values are now `u64`